src/Map.ts

```diff
@@ -345,24 +358,26 @@ export class Map extends maplibregl.Map {
       }
 
       // The attribution and logo must show when required
-      if ("logo" in tileJsonContent && tileJsonContent.logo) {
-        const logoURL: string = tileJsonContent.logo;
-
-        this.addControl(
-          new MaptilerLogoControl({ logoURL }),
-          options.logoPosition,
-        );
+      if (options.minimap !== true) {
```

Comment:
The comment above the option is `If set to true, the map will assume it is a minimap and forego the attribution control.` So it's only function when true is to ensure attribution is turned off. Otherwise it assumes options.

Reply:
Changing this but also making a different opinionated change by adding a `forceNoAttributionControl` flag to compensate.

Reply:
At this point, wouldn't it easier if the minimap was using a Maplibre map instead of the SDK map?